[2025-03-17T16:12:16.442+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-17T16:12:16.467+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_wether_dag.api scheduled__2025-03-16T21:00:00+00:00 [queued]>
[2025-03-17T16:12:16.480+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_wether_dag.api scheduled__2025-03-16T21:00:00+00:00 [queued]>
[2025-03-17T16:12:16.480+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-03-17T16:12:16.503+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): api> on 2025-03-16 21:00:00+00:00
[2025-03-17T16:12:16.511+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=64) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-17T16:12:16.513+0000] {standard_task_runner.py:72} INFO - Started process 114 to run task
[2025-03-17T16:12:16.513+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'api_wether_dag', 'api', 'scheduled__2025-03-16T21:00:00+00:00', '--job-id', '78', '--raw', '--subdir', 'DAGS_FOLDER/dag_wether.py', '--cfg-path', '/tmp/tmpr0q600t8']
[2025-03-17T16:12:16.517+0000] {standard_task_runner.py:105} INFO - Job 78: Subtask api
[2025-03-17T16:12:16.591+0000] {task_command.py:467} INFO - Running <TaskInstance: api_wether_dag.api scheduled__2025-03-16T21:00:00+00:00 [running]> on host c032966e105e
[2025-03-17T16:12:17.092+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='api_wether_dag' AIRFLOW_CTX_TASK_ID='api' AIRFLOW_CTX_EXECUTION_DATE='2025-03-16T21:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-16T21:00:00+00:00'
[2025-03-17T16:12:17.093+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-17T16:12:17.398+0000] {python.py:240} INFO - Done. Returned value was: ok
[2025-03-17T16:12:17.424+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-17T16:12:17.425+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=api_wether_dag, task_id=api, run_id=scheduled__2025-03-16T21:00:00+00:00, execution_date=20250316T210000, start_date=20250317T161216, end_date=20250317T161217
[2025-03-17T16:12:17.500+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-17T16:12:17.519+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-17T16:12:17.523+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
